cmd/bridge/main.go

```diff
@@ -229,6 +230,13 @@ func main() {
 		}
 	}
 
+	var i18nNamespaces []string
+	for _, str := range strings.Split(*fI18NamespacesFlags, ",") {
+		if str != "" {
```

Comment:
We shouldn't just ignore values like `,`. We should fail with an invalid flag value. This is almost certainly caused by a bug in our operator.

Reply:
but in this case the `str` wont contain `,` value, since we are splitting the `fI18NamespacesFlags` based on that character.

Reply:
Right and we're not treating it as an error. If I pass `,`, it will see an empty string and skip the value. It should fail validation.